tests/test_ext_intersphinx.py

```diff
@@ -229,3 +234,53 @@ def test_getsafeurl_unauthed():
     expected = 'https://domain.com/project/objects.inv'
     actual = _get_safe_url(url)
     assert expected == actual
+
+
+def test_debug_noargs(capsys):
+    """debug interface, without arguments"""
+    with pytest.raises(SystemExit):
+        debug(['sphinx/ext/intersphinx.py'])
+
+    expected = dedent("""
+        Print out an inventory file.
+        Error: must specify local path or URL to an inventory file.
+    """).strip()
+    stdout, stderr = capsys.readouterr()
+    assert stdout == ""
+    assert stderr == expected + "\n"
```

Comment:
I think it's important to assert that there is _some_ output, to let the user know how to use this tool. Is it a problem to have this assertion? If it doesn't much matter to you, I'd rather keep it.